src/web/lib/readerRevenueDevUtils.ts

```diff
@@ -132,7 +132,9 @@ const changeGeolocation = (
 		.then((current) => {
 			/* eslint-disable no-alert */
 			const geo = window.prompt(
-				`Enter two-letter geolocation code (e.g. GB, US, AU). Current is ${current}.`,
+				`Enter two-letter geolocation code (e.g. GB, US, AU). Current is ${
+					current || 'undefined'
+				}.`,
```

Comment:
We should not be designing in failure. If there's a use case where this is undefined then the message should speak to that in a useful way - eg. Please set your country etc.